core/EE_System.core.php

```diff
@@ -771,28 +701,22 @@ private static function _new_version_is_higher($activation_history_for_addon, $v
      *                                   sometimes containing 'unknown-date'
      * @return string
      */
-    private static function _get_most_recently_active_version_from_activation_history($activation_history)
+    private static function getMostRecentlyActiveVersion(array $activation_history): string
     {
-        $most_recently_active_version_activation = '1970-01-01 00:00:00';
+        $most_recent_activation_date  = '1970-01-01 00:00:00';
```

Comment:
mostly just renamed variables so that things are easier to read and therefore easier to understand